Document subscription vendor nicknaming (#134)
ClouDesire/docusaurus: docs/syndication.md

> If something could go wrong during the provisioning of a new tenant in your
application, discover a strategy to [handle exceptions](syndication.md#managing-provisioning-exceptions).

### Subscription naming

> You can customize the initial `name` of a subscription with:

Reviewer: ```suggestion
You can customize the initial `name` of a subscription that customers will find on their dashboard with:
```

You have now completed the standard workflow for a syndicated application,
congratulations!

### Subscription naming

> You can customize the `name` of a subscription by calling

Reviewer: ```suggestion
You can customize the initial `name` of a subscription with:
```

```http
PATCH /api/subscription/2388 HTTP/1.1

{
    "action": "SET_NAME",
    "name": "Nickname of this subscription"
}
```

> **Note** that if a customer names their subscription, the customer name takes
> precedence.

Reviewer: ```suggestion
**Note** that if the customer can rename their subscriptions and it takes precedence.
```
